main.py

```diff
@@ -43,6 +52,8 @@ def insert_table(df, tablename):
 
 def main():
     try:
+        remove_old_files()
```

Comment:
@susanxiong Is your thinking to do this at the start of the job to catch files for deletion if the job fails elsewhere? You could probably do this at the end without the need for the length checking.

Reply:
Yeah that is what i was thinking. That is a good point too.